packages/story-editor/src/components/library/common/searchInput.js

```diff
@@ -71,24 +66,24 @@ function WrappedSearchInput({
 
   const submitValue = useCallback(
     (value) => {
-      if (incremental && delayMs) {
+      if (delayMs) {
         changeSearchTermDebounced();
```

Comment:
Just removing the `incremental` check  completely changes the meaning of this.

We also need to remove `delayMs` and `changeSearchTermDebounced` completely, because they are only relevant for incremental search.

Basically, we can just remove `submitValue` and call `onSearch` directly